src/Simplex/Chat.hs

```diff
@@ -122,6 +122,7 @@ defaultChatConfig =
       testView = False,
       initialCleanupManagerDelay = 30 * 1000000, -- 30 seconds
       cleanupManagerInterval = 30 * 60, -- 30 minutes
+      cleanupManagerStepDelay = 3 * 1000000, -- 3 seconds
```

Comment:
why 3 seconds?

Reply:
arbitrary, to give other queries period when cleanup manager queries don't occupy database